README.md

```diff
@@ -32,26 +39,29 @@ Output:
 dtype: int32
 ```
 
-For additional examples, browse our complete [API documentation](https://docs.rapids.ai/api/cuml/stable/), or check out our more detailed [walkthrough notebooks](https://github.com/rapidsai/notebooks/tree/master/cuml).
+For additional examples, browse our complete [API
+documentation](https://docs.rapids.ai/api/cuml/stable/), or check out our more
+detailed [walkthrough
+notebooks](https://github.com/rapidsai/notebooks/tree/master/cuml).
 
-### Supported Algorithms:
 
-| Algorithm | Scale | Notes |
+### Supported Algorithms
+| Category | Algorithm | Notes |
 | --- | --- | --- |
-| Coordinate Descent | Single-GPU | |
-| Density-Based Spatial Clustering of Applications with Noise (DBSCAN) | Single GPU |
-| K-Means Clustering | Single-GPU |
-| K-Nearest Neighbors (KNN) | Multi-GPU with [dask-cuml](http://github.com/rapidsai/dask-cuml) <br> Uses [Faiss](https://github.com/facebookresearch/faiss) |
-| Linear Kalman Filter | Single-GPU |
-| Linear Regression (OLS) | Single GPU | Multi-GPU available in conda cuda10 package and [dask-cuml](http://github.com/rapidsai/dask-cuml) |
-| Linear Regression with Lasso Regularization | Single-GPU |
-| Linear Regression with Elastic-Net Regularization | Single-GPU |
-| Principal Component Analysis (PCA) | Single GPU |
-| Ridge Regression | Single-GPU |
-| Stochastic Gradient Descent | Single-GPU | for linear regression, logistic regression, and linear svm with L1, L2, and elastic-net penalties |
-| Truncated Singular Value Decomposition (tSVD) | Single GPU | Multi-GPU available in conda cuda10 package |
-| UMAP | Single-GPU |
-
+| **Clustering** |  Density-Based Spatial Clustering of Applications with Noise (DBSCAN) | |
+|  | K-Means | |
+|  | K-Nearest Neighbors (KNN) | Multi-GPU with [dask-cuml](http://github.com/rapidsai/dask-cuml) <br> Uses [Faiss](https://github.com/facebookresearch/faiss) |
+| **Dimensionality Reduction** | Principal Components Analysis (PCA) | |
+| | Truncated Singular Value Decomposition (tSVD) | Multi-GPU version available (CUDA 10 only) |
+| | Uniform Manifold Approximation and Projection (UMAP) | |
+| | Random Projection | |
+| **Linear Models for Regression or Classification** | Linear Regression (OLS) | Multi-GPU available in conda CUDA 10 package and [dask-cuml](http://github.com/rapidsai/dask-cuml) |
+| | Linear Regression with Lasso or Ridge Regularization | |
+| | ElasticNet Regression | |
+| | Logistic Regression | |
+| | SGD, LBFGS, and Coordinate Descent Solvers for linear models | |
```

Comment:
it is a very small thing, i think having the expanded form of SGD and LBFGS might help the readers

Reply:
For consistency indeed they should be 

```suggestion
| | Stochastic Gradient Descent (SGD), Quasi-Newton (QN) including L-BFGS and OWL-QN, and Coordinate Descent (CD) Solvers for linear models | |
```